From 5ca0684c50a9fb8257f1d2a2d20ab63120013ee3 Mon Sep 17 00:00:00 2001
From: dljames <dljames@chromium.org>
Date: Fri, 23 Jan 2026 12:26:31 -0800
Subject: [PATCH] [Vertical Tabs] Fix crash when dragging from inactive Vertical TabStrip

Fixes a crash that would happen if a Tab was being dragged while the
Vertical TabStrip was in an inactive / hidden state.

A regression test is not added since the exact reproduction steps for
this have not been found yet.

Change-Id: I8e254073d016d5200566c70c2b4b229df5624c1c
Bug: 478217455
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/7514288
Reviewed-by: Christopher Grant <cjgrant@chromium.org>
Commit-Queue: Darryl James <dljames@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1573883}
---

--- a/chrome/browser/ui/views/frame/vertical_tab_strip_region_view.cc
+++ b/chrome/browser/ui/views/frame/vertical_tab_strip_region_view.cc
@@ -514,6 +514,10 @@ bool VerticalTabStripRegionView::IsFrame
 
 TabDragTarget* VerticalTabStripRegionView::GetTabDragTarget(
     const gfx::Point& point_in_screen) {
+  if (!drag_handler_) {
+    return nullptr;
+  }
+
   VerticalUnpinnedTabContainerView* container = GetUnpinnedTabsContainer();
   CHECK(container);
   if (container->GetBoundsInScreen().Contains(point_in_screen)) {
